src/components/resource/_fields/coverage-statement/coverage-statement-fields.js

```diff
@@ -5,10 +5,12 @@ import { RadioButton, TextArea } from '@folio/stripes/components';
 import { FormattedMessage } from 'react-intl';
 import styles from './coverage-statement-fields.css';
 
-function validate(value, { hasCoverageStatement }) {
+import { COVERAGE_STATEMENT_VALUE_MAX_LENGTH } from '../../../../constants';
+
+const validate = (value, { hasCoverageStatement }) => {
   let error;
 
-  if (value && value.length > 350) {
+  if (value && value.length > COVERAGE_STATEMENT_VALUE_MAX_LENGTH) {
```

Comment:
Please, check if it's working
```suggestion
  if (value?.length > COVERAGE_STATEMENT_VALUE_MAX_LENGTH) {
```